### ISSUE TITLE
signConvention does not use two's complement?

### ISSUE THREAD

**Issue by ndantam** [hubo/hubo-ach#50]
It looks like signConvention() and DrcSignConvention converted a 24-bit two's complement signed integer to a 24-bit signed integer using a sign bit.   Do the Hubo JMC's really use sign bits instead of two's complement?

**Comment by robEllenberg** [comment]
Yeah...we had that same facepalm moment back when we first started with
jaemi.

**Comment by ndantam** [comment]
Wow.

Well, that at least deserves some comments in the code.

**Comment by thedancomplex** [comment]
yes "facepalm" exactly.

On Fri, Nov 1, 2013 at 12:32 AM, Neil T. Dantam notifications@github.comwrote:

> Wow.
> 
> Well, that at least deserves some comments in the code.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/hubo/hubo-ach/issues/50#issuecomment-27547230
> .

## 

Daniel M. Lofaro Ph.D
Assistant Professor, George Mason University
Electrical and Computer Engineering Department
DARPA Robot Challenge Track-A Team DRC-Hubo Researcher
e: dan@danLofaro.com
w: http://danLofaro.com http://danlofaro.com/
